config/src/main/java/org/springframework/security/config/annotation/web/configurers/CsrfCustomizer.java

```diff
@@ -0,0 +1,57 @@
+package org.springframework.security.config.annotation.web.configurers;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.springframework.security.config.Customizer;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.web.csrf.CookieCsrfTokenRepository;
+import org.springframework.security.web.csrf.CsrfToken;
+import org.springframework.security.web.csrf.CsrfTokenRequestAttributeHandler;
+import org.springframework.security.web.csrf.CsrfTokenRequestHandler;
+import org.springframework.security.web.csrf.XorCsrfTokenRequestAttributeHandler;
+import org.springframework.util.StringUtils;
+
+import java.util.function.Supplier;
+
+public class CsrfCustomizer {
+	public static Customizer<CsrfConfigurer<HttpSecurity>> spaDefaults() {
```

Comment:
Please add a JavaDoc that explains what the method does and includes a `@since 7.0` attribute.